examples/Application/Traffic-Light-Control/replay_buffer.py

```diff
@@ -0,0 +1,100 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Can we use the parl.utils.ReplayMemory directly?

Reply:
Yes, fixed.